src/app/routes/home/components/loan-thumb/loan-thumb.component.ts

```diff
@@ -0,0 +1,15 @@
+import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { Models } from '../../../../shared/models';
```

Comment:
Minor, import models from the "$models" path.

Reply:
Guessing you mean `import { Models } from '$shared'`